src/Client/People.php

```diff
@@ -20,7 +21,12 @@
 final class People implements Iterator, Sequence
 {
     use ArrayFromIterator;
-    use SlicedIterator;
+    use SlicedArrayAccess;
```

Comment:
looks like this is going to be repeated lots of times, is there a way to compose a trait from `SlicedArrayAccess` and `SlicedIterator` that keeps the conflict resolution in one place?